app/javascript/components/main-menu/second-level.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MenuItem } from './main-menu';
+import { menuProps } from './recursive-props';
+
+const SecondLevel = ({
+  id,
+  title,
+  href,
+  items,
+  level,
+}) => (
+  <li className={`list-group-item ${items.length > 0 && 'tertiary-nav-item-pf'}`} data-target={`#menu-${id}`}>
```

Comment:
This will add `false` to className if the condition is not met. We don't want that.